patches/tModLoader/Terraria/PopupText.cs.patch

```diff
@@ -17,6 +17,48 @@
  		public bool sonar;
  		public PopupTextContext context;
  		public int npcNetID;
+@@ -40,6 +_,41 @@
+ 			}
+ 		}
+ 
```

Comment:
Is this new NewText overload necessary? If it really is, make PopupText partial, add a new file called PopupText.tML.cs and add the method there

Reply:
PopupText.NewText with a string allows for modders to include any text they want instead of the Item or NPC name text options.